testdata/valid/07-deleteall.diff

```diff
@@ -1,19 +1,76 @@
 Filters:
 --- Current
 +++ TO BE APPLIED
-@@ -1,15 +1 @@
+@@ -1,72 +1 @@
```

Comment:
It seems like it's working as intended, but I'll leave this here just to double check. Are we sure that the diff will not cut out parts of the criteria? I'm thinking about cases where the diff is only in one element of a long list. If only a few elements are displayed, and not the full filter, it may become less clear?

Reply:
I guess that depends on the behavior of difflib. WDYT?

Reply:
Perhaps it makes sense that `indent` will also show the unindented form (if indentation was performed)?